src/scripts/main.js

```diff
@@ -1,5 +1,27 @@
 'use strict';
 
-// const pushNotification = (top, right, title, description, type) => {
-//   // write code here
-// };
+const pushNotification = (top, right, title, description, type) => {
+  const block = document.createElement('div');
+
+  block.className = 'notification';
+  document.body.append(block);
+
+  block.style.top = `${top}px`;
+  block.style.right = `${right}px`;
+  block.classList.add(type);
```

Comment:
try this block.classList.add('notification', type);

Reply:
Done